db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Team.create(name: 'Arizona Cardinals', short_name: 'Arizona', code: 'ARI')
+Team.create(name: 'Atlanta Falcons', short_name: 'Atlanta', code: 'ATL')
+Team.create(name: 'Baltimore Ravens', short_name: 'Baltimore', code: 'BAL')
+Team.create(name: 'Buffalo Bills', short_name: 'Buffalo', code: 'BUF')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.